internal/pkg/service/common/task/node.go

```diff
@@ -254,7 +254,7 @@ func (n *Node) runTask(logger log.Logger, task Task, cfg Config) {
 	}
 
 	// Update telemetry
-	errType := errorType(result.Err())
+	errType := telemetry.ErrorType(result.Err())
```

Comment:
Vo viacerych metrikach chcem mat `error_type`: `net`, `context_deadline`, `other`, .... aby sa dalo jednoducho vyreslit o co sa jedna, a napr. `context_cancelled` ignorovat (zrusenie HTTP requestu klientom).

Kedze je to potrebne v roznych pkgs, tak som tento helper presunul do `telemetry` pkg.